TrashMobMobile/ViewModels/CreateEventViewModelNew.cs

```diff
@@ -224,6 +238,8 @@ public void SetCurrentStep(StepType step)
                 SetCurrentView();
             }
         }
+
+        CanGoBack = CurrentStep != 0 && CurrentStep != 4;
```

Comment:
Step 1 - canGoBack false
Step 2 - canGoBack true
Step 3 - canGoBack true
Step 4 - canGoBack true (until save has been hit)
Step 5+ - canGoBack false

Reply:
I asume that save action goes to next step